File: src/main/java/olivermakesco/de/refmagic/item/WitheriteArmorItem.java
package olivermakesco.de.refmagic.item;

import net.minecraft.entity.EquipmentSlot;
import net.minecraft.item.ArmorItem;
import olivermakesco.de.refmagic.material.WitheriteArmorMaterial;

public class WitheriteArmorItem extends ArmorItem {
    public WitheriteArmorItem(EquipmentSlot equipmentSlot, Settings settings) {
        super(WitheriteArmorMaterial.instance, equipmentSlot, settings);
    }
}

File: src/main/kotlin/olivermakesco/de/refmagic/item/WitheriteArmorItem.kt
package olivermakesco.de.refmagic.item

import net.minecraft.entity.EquipmentSlot
import net.minecraft.item.ArmorItem
import olivermakesco.de.refmagic.material.WitheriteArmorMaterial

class WitheriteArmorItem(equipmentSlot: EquipmentSlot?, settings: Settings?) :
    ArmorItem(WitheriteArmorMaterial.instance, equipmentSlot, settings)